transform_inline.ml

```diff
@@ -22,6 +12,9 @@ type inlining_candidate = {
 }
 and inlining_site = inlining_candidate list
 
+let as_inlining_input (func : afunction) =
```

Comment:
This is basically `Analysis.as_analysis_input` except that it uses the active version, rather than a given version.